pkg/setting/setting.go

```diff
@@ -764,6 +767,13 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 	dashboards := iniFile.Section("dashboards")
 	DashboardVersionsToKeep = dashboards.Key("versions_to_keep").MustInt(20)
 
+	// read dashboard settings
+	annotations := iniFile.Section("annotations")
+	DaysToKeepAnnotations = annotations.Key("days_to_keep").MustInt(60)
+	if DaysToKeepAnnotations < 5 {
```

Comment:
Any thoughts about this? Why not below 5? :)

Reply:
I'm actually unsure about this too @bergquist - 5 was the minimum before my revising the PR.